src/platform/unix/mod.rs

```diff
@@ -982,6 +970,13 @@ unsafe fn create_shmem(name: *const c_char, length: usize) -> c_int {
     fd
 }
 
+#[cfg(target_os="android")]
+unsafe fn create_shmem(name: *const c_char, length: usize) -> c_int {
+    let fd = ashmem_create_region(name, length);
```

Comment:
**NB:** I have not tested the `android` code yet. This is more or less a placeholder for now.